open_ce/inputs.py

```diff
@@ -412,6 +424,7 @@ def parse_args(parser, arg_strings=None):
 
     if "container_build" not in vars(args).keys() or not args.container_build:
         _check_ppc_arch(args)
+        _check_cuda_versions(args)
 
```

Comment:
This new function should not be inside if block at line 425. It should be out of if condition.

Reply:
Corrected place of function